frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -11,8 +11,11 @@ import {
 
 type NoBucket = null;
 
-export type BucketListItem = Lib.BucketDisplayInfo & {
+export type BucketListItem = {
+  displayName: string;
   bucket: Lib.Bucket | NoBucket;
+  default?: boolean;
+  selected?: boolean;
 };
```

Comment:
Why can't we extend `Lib.BucketDisplayInfo` here anymore?

Reply:
Because I added new required `shortName` prop and here https://github.com/metabase/metabase/blob/0f606d40d88a886220decfa42c20252e01f06126/frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BinningStrategyPickerPopover.tsx#L25 we add items that are not coming from `displayInfo`